app/views/dashboards/show.html.erb

```diff
@@ -1,19 +1,19 @@
 <!-- show.html.erb -->
-<h1>Welcome, <%= @user.email %></h1>
+<h1>Welcome, <%= current_user.email %></h1>
 
-<%# <% if @user.details_completed? %> %>
-  <!-- Display user's plans and other content -->
-  <h2>Your Weekly Plans</h2>
-  <% @plans.each do |plan| %>
-    <p><%= plan.name %></p>
-    <!-- Display any other relevant plan information -->
-  <% end %>
-  <%# <h2>Day Plans</h2> %>
-  <%# <% @days.each do |day| %> %>
-    <%# <h3><%= link_to day.day.strftime("%A, %B %d, %Y"), weekly_plan_day_plan_path(day.weekly_plan, day) %></h3> %>
-    <!-- Display any other relevant day information -->
-  <%# <% end %> %>
-<%# <% else %> %>
-  <!-- Display a button to generate first plans -->
-  <%= link_to 'Generate First Plans', new_weekly_plan_path(@user), class: 'btn btn-primary' %>
-<%# <% end %> %>
+<!-- Display user's plans and other content -->
+<h2>Your Weekly Plans</h2>
+<% @weekly_plans.each do |plan| %>
+  <p><%= plan.name %></p>
+  <p><%= link_to weekly_plan.name, weekly_plan_path(weekly_plan) %></p>
+  <!-- Display any other relevant plan information -->
+<h2>Day Plans for <%= plan.name %></h2>
+<% plan.day_plans do |day_plan| %>
+  <h3><%= link_to day_plan.day.day_number, weekly_plan_day_plan_path(day_plan.weekly_plan, day_plan) %></h3>
```

Comment:
since the day_plan has the reference to the weekly plan it's not necessary top nest the route for the day_plan show. day_plan_path(day_plan) is enough.